onnxruntime/core/providers/js/operators/pad.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/js/js_kernel.h"
+#include "core/providers/cpu/tensor/padbase.h"
+
+namespace onnxruntime {
+namespace js {
+
+class Pad : public JsKernel, public PadBase {
+ public:
+  Pad(const OpKernelInfo& info) : JsKernel(info), PadBase(info) {
+    std::vector<int32_t> pads;
+    if (!is_dynamic_) {
+      pads.resize(pads_.size());
+      for (size_t i = 0; i < pads_.size(); ++i) {
+        pads[i] = gsl::narrow_cast<int32_t>(pads_[i]);
+      }
+    }
+
+    JSEP_INIT_KERNEL_ATTRIBUTE(Pad, ({"mode" : $1,
+                                      "value" : $2,
+                                      "pads" : $3 ? Array.from(HEAP32.subarray($5, $5 + $4)) : []}),
```

Comment:
seems no need to pass is_dymanic. check if size is zero is enough

Reply:
Done

Reply:
I mean, we can decrease the 3 parameters into 2. the one below `pads.size() > 0` is no need.